IONOS Cloud Object Storage (Berlin).cyberduckprofile

```diff
@@ -0,0 +1,2281 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+    <dict>
+        <key>Protocol</key>
+        <string>s3</string>
+        <key>Vendor</key>
+        <string>ionos-berlin</string>
+        <key>Description</key>
+        <string>IONOS Cloud Object Storage (Berlin, Germany)</string>
+        <key>Default Hostname</key>
+        <string>s3-eu-central-2.ionoscloud.com</string>
+        <key>Default Port</key>
+        <string>443</string>
+        <key>Scheme</key>
+        <string>https</string>
+        <key>Hostname Configurable</key>
+        <false/>
+        <key>Port Configurable</key>
+        <false/>
+        <key>Username Placeholder</key>
+        <string>Access Key</string>
+        <key>Password Placeholder</key>
+        <string>Secret Key</string>
+        <key>Authorization</key>
+        <string>AWS4HMACSHA256</string>        
+        <key>Region</key>
+        <string>eu-central-2</string>        
+        <key>Regions</key>
+        <array>
+            <string>eu-central-2</string>
+        </array>
+        <key>Properties</key>
+        <array>
+            <string>s3.storage.class.options=STANDARD</string>
+            <string>s3service.disable-dns-buckets=true</string>
```

Comment:
Please change this to `s3.bucket.virtualhost.disable=true` as this has changed with iterate-ch/cyberduck@c8c85d2c

Reply:
@dkocher I updated this according to your request.